helpers/trasladosHelper/trasladosHelper.go

```diff
@@ -0,0 +1,319 @@
+package trasladoshelper
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/astaxie/beego"
+	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/arka_mid/helpers/utilsHelper"
+	"github.com/udistrital/arka_mid/models"
+	"github.com/udistrital/utils_oas/request"
+)
+
+// GetDetalle Consulta los funcionarios, ubicación y elementos asociados a un traslado
+func GetDetalleTraslado(id int) (Traslado map[string]interface{}, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{"funcion": "/GetDetalleTraslado", "err": err, "status": "502"}
+			panic(outputError)
+		}
+	}()
+
+	var (
+		urlcrud    string
+		movimiento models.Movimiento
+		detalle    models.DetalleTraslado
+	)
+	Traslado = make(map[string]interface{})
+
+	// Se consulta el movimiento
+	urlcrud = "http://" + beego.AppConfig.String("movimientosArkaService") + "movimiento/" + strconv.Itoa(id)
+	if err := request.GetJson(urlcrud, &movimiento); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetDetalleTraslado - request.GetJson(urlcrud, &movimiento)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+
+	if err := json.Unmarshal([]byte(movimiento.Detalle), &detalle); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetDetalleTraslado - json.Unmarshal([]byte(movimiento.Detalle), &detalle)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+
+	// Se consulta el detalle del funcionario origen
+	if origen, err := GetDetalleFuncionario(detalle.FuncionarioOrigen); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetDetalleTraslado - GetDetalleFuncionario(detalle.FuncionarioOrigen)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
```

Comment:
(En general) Cuando ya se tenga un error formateado. ej, que un helper ya retorna el error formateado, retornarlo directamente, no es necesario encapsularlo
```suggestion
		return nil, err
```